src/components/Composer.vue

```diff
@@ -15,82 +16,165 @@
 				:placeholder="t('mail', 'Select account')"
 				:clear-on-select="false"
 				@select="onAliasChange" />
+				<button 
+					title="" 
+					:class="['icon-more',{'active':!autoLimit}]"
+					@click.prevent="autoLimit = !autoLimit">
+				</button>
+			</div>
 		</div>
 		<div class="composer-fields">
 			<label class="to-label" for="to">
 				{{ t('mail', 'To') }}
 			</label>
-			<Multiselect
-				id="to"
-				ref="toLabel"
-				v-model="selectTo"
-				:options="selectableRecipients"
-				:taggable="true"
-				label="label"
-				track-by="email"
-				:limit="4"
-				:multiple="true"
-				:placeholder="t('mail', 'Contact or email address …')"
-				:clear-on-select="true"
-				:close-on-select="false"
-				:show-no-options="false"
-				:preserve-search="true"
-				:hide-selected="true"
-				:loading="loadingIndicatorTo"
-				@input="callSaveDraft(true, getMessageData)"
-				@tag="onNewToAddr"
-				@search-change="onAutocomplete($event, 'to')" />
-			<a v-if="!showCC"
-				class="copy-toggle"
-				href="#"
-				@click.prevent="showCC = true">
-				{{ t('mail', '+ Cc/Bcc') }}
-			</a>
+			<div class="composer-fields--custom">
+				<Multiselect :class="{'opened': !autoLimit}"
+					id="to"
+					ref="toLabel"
+					v-model="selectTo"
+					:options="selectableRecipients"
+					:taggable="true"
+					label="label"
+					track-by="email"
+					:multiple="true"
+					:placeholder="t('mail', 'Contact or email address …')"
+					:clear-on-select="false"
+					:close-on-select="false"
+					:show-no-options="false"
+					:preserve-search="true"
+					:hide-selected="true"
+					:loading="loadingIndicatorTo"
+					v-bind:auto-limit="autoLimit"
+					@input="callSaveDraft(true, getMessageData)"
+				    @tag="onNewToAddr"
+				    @search-change="onAutocomplete($event, 'to')">
+					<template #tag="{ option }">
+						<div class="multiselect__tag multiselect__tag-custom">
+							<ListItemIcon
+								:no-margin="true"
+								:title="option.label"
+								:avatar-size="24" />
+							<Actions>
+								<ActionButton
+									icon="icon-close"
+									@click.prevent="removeRecipientTo(option)"
+								/>
+							</Actions>
+						</div>
+					</template>
+					<template #option="{ option }">
+						<div class="multiselect__tag multiselect__tag-custom">
+							<ListItemIcon
+								:no-margin="true"
+								:title="option.label"
+								:subtitle="option.email"
+								:avatar-size="24" />
+						</div>
+					</template>
+				</Multiselect>
+				<button 
+					:title="t('mail', '+ Cc/Bcc')" 
+					:class="['icon-group',{'active':showCC}]"
+					@click.prevent="showCC = !showCC">
+				</button>		
```

Comment:
Please hide the button once it was clicked and the selection was expanded (like it was before). Otherwise, important information (cc, bcc) is hidden and the user might think they are empty.

```js
v-if="showCc"
```

Reply:
Hi! Yes, it makes sense. I'll fix it!